api/pom.xml

```diff
@@ -10,7 +10,7 @@
     <parent>
         <groupId>com.flipkart.poseidon</groupId>
         <artifactId>poseidon</artifactId>
-        <version>5.11.0-SNAPSHOT</version>
+        <version>5.11.0-PX-SNAPSHOT</version>
```

Comment:
Please revert all the pom file changes for using PX snapshot. I will update the snapshot at 5.11.0 in clojars and once tested, I will release 5.11.0